doc/api/process.md

```diff
@@ -1775,6 +1784,317 @@ Will generate an object similar to:
   tz: '2016b' }
 ```
 
+## Performance Timing API
+<!-- YAML
+added: REPLACEME
+-->
+
+The Performance Timing API provides an implementation of the
+[W3C Performance Timeline][] specification. The purpose of the API
```

Comment:
@domenic Should we link to the published version of the spec or the latest editor's draft?

Also this should reference User Timing and High Resolution Time specs since we also implement those.